design/run_and_collect_data.md

```diff
@@ -0,0 +1,27 @@
+#run.py proposal
+
+##objectives
+
+Python will be used to run all instances of scp residing in /test, then create an executive summary in csv which can be used to quickly create charts for the report 
+
+##plan
+
+	1.	run.py will will load each instance of scp provided - residing in /test_data
+
+	2.	for each instance all heuristics will be run (assuming this will be 6 - but will begin with just the one for now)
```

Comment:
Will this approach handle multiple runs of the same heuristic against the same data set. Does each run need a unique id?